lib/auth/oauth_gems.rb

```diff
@@ -226,5 +226,10 @@ def authorize_params
       end
     end
   end
+
+  def callback_url
+    options[:redirect_uri] || (full_host + script_name + callback_path)
```

Comment:
nitpick, should call `super` instead of `(full_host + script_name + callback_path)`